File: src/main/java/com/dochiri/hexagonal/HexagonalApplication.java
package com.dochiri.hexagonal;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.boot.context.properties.ConfigurationPropertiesScan;

@SpringBootApplication
@ConfigurationPropertiesScan
public class HexagonalApplication {

    static void main(String[] args) {
        SpringApplication.run(HexagonalApplication.class, args);
    }

}

File: src/main/kotlin/com/dochiri/hexagonal/HexagonalApplication.kt
package com.dochiri.hexagonal

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.context.properties.ConfigurationPropertiesScan
import org.springframework.boot.runApplication

@SpringBootApplication
@ConfigurationPropertiesScan
class HexagonalApplication

fun main(args: Array<String>) {
    runApplication<HexagonalApplication>(*args)
}
